### ISSUE TITLE
wait_for

### ISSUE THREAD

**Issue by 0xn3bs** [selenium-cucumber/selenium-cucumber-ruby#22]
Would there be side-effects to wrapping most canned-in methods with wait_for?

**Comment by Carlbc18** [comment]
I don't believe so, but the driver can be configured to implicitly wait (which is the OOTB default) by setting  $driver.manage.timeouts.implicit_wait = <time in seconds> or by using the explicit waiter which is available and can easily be configured w/n your support env.rb.  What is the purpose for using wait_for?

**Comment by sameer49** [comment]
@Carlbc18 I guess we can add wait_for in each method in set time as global variable which can be modified in between the test.
